Update lg_04_labels.md (#1040)
ocaml/v3.ocaml.org: data/tutorials/lg_04_labels.md

```

(Notice that both `to` and `end` are reserved words in OCaml, so they

> cannot be used as labels. This means you cannot have `~from/~to` or

Reviewer: This is confusing (no issue for `~from` and `~start`), you just cannot have `~to` and `~end` but you can have `~to_` and `~end_` for example.

Author: Thank you for your comment, @gpetiot. Shall I change it to `~from/~to_`, etc?

Reviewer: The example is fine with `~first` and `~last`. I would just fix the note: "This means you cannot have `~to` or `~end`".

Author: Changed! Please double check I did it correctly.

Reviewer: All good!

- : window = {title = "My Application"; width = 640; height = 480}
```

So far, this is just the imperative "mutable records" that we talked

> about in the [previous chapter](https://ocaml.org/docs/if-statements-and-loops). Now the complex part is the `open_window`

Reviewer 2: ```suggestion
about in ["If Statements, Loops and Recursions"](https://ocaml.org/docs/if-statements-and-loops). Now the complex part is the `open_window`
```

Docs aren't a book. There is no order between them.

### `Warning: This optional argument cannot be erased`
We've just touched upon labels and optional arguments, but even this
brief explanation might have raised several questions. The first may be

> why use the extra `unit` argument to `open_window`. Let's try defining this

Reviewer: I had to re-read this sentence a few times, could we quote the question:

The first may be:
> Why use the extra `unit` argument to `open_window`?

Let's try defining this function without the extra `unit`:

Author: I like that!

> "Why use the extra unit argument to open_window?" Let's try defining this

Reviewer 2: ```suggestion
"Why use the extra `()` (unit) argument to `open_window`?" Let's try defining this
```
This is syntax

The declaration `~foo:foo` creates a variable called simply `foo`, so if
you need the value just use plain `foo`.

> However, thing get complicated for two reasons: first, the shorthand

Reviewer: "thing" -> "things"

Author: Nice catch! Done!

  close (fd);                                            // line 15
}
```

> When I compile the code, I get a whole bunch of errors including:

Reviewer: Not directly a comment on this PR, for future ref. It was "we" in the beginning of the page and now switched to "I", maybe we should open another PR to fix these inconsistencies.

Reviewer 3: I think you're right. Consistency in the voice is desirable. I opened https://github.com/ocaml/ocaml.org/issues/1154.

Author: Consistency is key!

I responded in #1154, so once we decide overall, I will do that moving forward and re-address the ones I've already done.

Author: I removed the other first person singular instances in this document.
